src/widgets/hits/hits.js

```diff
@@ -41,7 +44,11 @@ function hits({
     hitsPerPage = 20
   } = {}) {
   if (!container) {
-    throw new Error(usage);
+    throw new Error('Must provide a container.' + usage);
+  }
+
+  if (templates.item && templates.items) {
+    throw new Error('Must contain only items OR item template.' + usage);
```

Comment:
Its allItems I believe but that made me think, what's the best API entry to express this new option? allItems? items? full?